cmd/balloons/nri-resmgr-balloons-deployment.yaml.in

```diff
@@ -107,20 +107,32 @@ spec:
         hostPath:
           path: /var/lib/nri-resmgr
           type: DirectoryOrCreate
-      - name: nri-resmgr
         hostPath:
           path: /var/lib/nri-resmgr
       - name: hostsysfs
         hostPath:
           path: /host
           type: DirectoryOrCreate
-      - name: sys
         hostPath:
           path: /sys
           type: Directory
```

Comment:
Is this (removing the `name`s) intentional ? Doesn't that leave us with a single entry that has several `hostPath`s of which only the last one being effective ?